Enterprise-CMCS/macpro-mdct-mcr
# [MDCT-2822] update with new rounding field

### Description



### Related ticket(s)


MDCT-2822

---
### How to test

- [Login](https://d1pq7j5x6f2dig.cloudfront.net/)
- For both MCPAR and MLR, confirm that rounding is matching with [rounding sheet](https://docs.google.com/spreadsheets/d/1j6NdmOfIQ09Zb3JWFXXADNcXUhpD-ZVpYydjlPVDPec/edit#gid=0)

### Important updates




---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
